### PR TITLE
add more perf logs

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs [hunk 1]
```diff
@@ -209,14 +212,17 @@ select new
...
             {
-                var hostService = creator.CreateHostService(
-                    parameters,
-                    templateProcessor,
-                    MarkdownService,
-                    MetadataValidators,
-                    pair.processor,
-                    pair.item?.Select(f => f.file));
-                yield return hostService;
+                using (new LoggerPhaseScope(pair.processor.Name, true))
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs]: >pair.processor.Name [](start = 44, length = 19)

need we add some verb here?

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs (hunk 1 above)
Comment (ansyral) [inline src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs]: already have a `Load` prefix

**Pull request merged**